client/signup/steps/initial-intent/index.tsx

```diff
@@ -27,7 +28,41 @@ export default function InitialIntentStep( props: Props ) {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [] );
 
+	const getRedirectForAnswers = ( _answerKeys: string[] ): string => {
+		if ( _answerKeys.includes( 'migrate-or-import-site' ) ) {
+			return `/setup/${ IMPORT_HOSTED_SITE_FLOW }`;
+		}
+
+		if ( _answerKeys.includes( 'client' ) ) {
+			// return `/setup/${ IMPORT_HOSTED_SITE_FLOW }`;
```

Comment:
@escapemanuele for some reason I thought that we needed to redirect to a separate flow for agencies. Will delete it

Reply:
No, only migration, difm and newsletter!